### ISSUE TITLE
"DiscordInteraction" doesn't work in Direct Messages

### ISSUE THREAD

**Issue by MrPowerGamerBR** [kordlib/kord#190]
On a direct message, the `member` and `guild_id` are null.

**Guild:**
```json
{"channel_id":"513405772911345664","data":{"id":"812846584084955136","name":"online"},"guild_id":"268353819409252352","id":"814909719054647316","member":{"deaf":false,"is_pending":false,"joined_at":"2017-01-10T12:22:16.323000+00:00","mute":false,"nick":null,"pending":false,"permissions":"4294967295","premium_since":null,"roles":["297051132793061378","334711955736625185","401353261266763778","650869752524439563","693210200496144424"],"user":{"avatar":"8bd2b747f135c65fd2da873c34ba485c","discriminator":"4185","id":"123170274651668480","public_flags":131712,"username":"MrPowerGamerBR"}},"token":"-","type":2,"version":1}
```

**Direct Message:**
```json
{"channel_id":"637662290170216458","data":{"id":"812857383717699626","name":"online"},"id":"814909846746431539","token":"-","type":2,"user":{"avatar":"8bd2b747f135c65fd2da873c34ba485c","discriminator":"4185","id":"123170274651668480","public_flags":131712,"username":"MrPowerGamerBR"},"version":1}
```

This causes errors when trying to parse a command received via direct message.

**Comment by DRSchlaubi** [comment]
Slash commands don't support Direct messages in general so this works as intended. How do you even get an interaction event from DMs?

**Comment by MrPowerGamerBR** [comment]
> Slash commands don't support Direct messages in general so this works as intended. How do you even get an interaction event from DMs?

[Global Commands are available via Direct Messages since 4th of Feb](https://github.com/discord/discord-api-docs/issues/2490#issuecomment-773626150)

**Comment by DRSchlaubi** [comment]
Look at that Discord updates their API so quickly I didn't even notice

**Comment by DRSchlaubi** [comment]
Referencing discord PR here:  discordapp/discord-api-docs#2568

**Comment by HopeBaron** [comment]
fixed as of #201
